<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BoundaryConditionsForm</class>
 <widget class="QWidget" name="BoundaryConditionsForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>281</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBoxMain">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Boundary conditions</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="9" column="0">
       <widget class="BoundaryConditionsTurbulenceInletView" name="turbulenceWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="BoundaryConditionsRoughWallView" name="roughWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="BoundaryConditionsMeteoView" name="meteoWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="BoundaryConditionsMobileMeshView" name="mobileMeshWidget" native="true"/>
      </item>
      <item row="12" column="0">
       <widget class="BoundaryConditionsWallRadiativeTransferView" name="radiativeWidget" native="true"/>
      </item>
      <item row="10" column="0">
       <widget class="BoundaryConditionsScalarsView" name="scalarsWidget" native="true"/>
      </item>
      <item row="5" column="0">
       <widget class="BoundaryConditionsVelocityInletView" name="velocityWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="BoundaryConditionsSlidingWallView" name="slidingWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="BoundaryConditionsCoalInletView" name="coalWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="BoundaryConditionsElectricalView" name="electricalWidget" native="true"/>
      </item>
      <item row="14" column="0">
       <widget class="BoundaryConditionsExternalHeadLossesView" name="externalHeadLossesWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="BoundaryConditionsHydraulicHeadView" name="hydraulicheadWidget" native="true"/>
      </item>
      <item row="16" column="0">
       <widget class="BoundaryConditionsPressureView" name="pressureWidget" native="true"/>
      </item>
      <item row="4" column="0">
       <widget class="BoundaryConditionsMappedInletView" name="mappedInletWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="BoundaryConditionsConvectiveInletView" name="convectiveInletWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="BoundaryConditionsCompressibleOutletView" name="compressibleOutletWidget" native="true">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>16</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <widget class="LagrangianBoundaryView" name="particlesWidget" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>BoundaryConditionsRoughWallView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsRoughWallView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsSlidingWallView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsSlidingWallView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsVelocityInletView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsVelocityInletView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsMappedInletView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsMappedInletView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsCoalInletView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsCoalInletView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsExternalHeadLossesView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsExternalHeadLossesView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsCompressibleOutletView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsCompressibleOutletView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsTurbulenceInletView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsTurbulenceInletView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsMeteoView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsMeteoView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsScalarsView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsScalarsView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsMobileMeshView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsMobileMeshView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsWallRadiativeTransferView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsWallRadiativeTransferView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsElectricalView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsElectricalView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsHydraulicHeadView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsHydraulicHeadView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsPressureView</class>
   <extends>QWidget</extends>
   <header location="global">BoundaryConditionsPressureView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoundaryConditionsConvectiveInletView</class>
   <extends>QWidget</extends>
   <header>BoundaryConditionsConvectiveInletView.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LagrangianBoundaryView</class>
   <extends>QWidget</extends>
   <header location="global">LagrangianBoundaryView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
